crates/pacdef_core/src/backend/actual/python.rs

```diff
@@ -57,9 +70,25 @@ fn run_pip_command(args: &[&str]) -> Result<Value> {
 impl Python {
     pub(crate) fn new() -> Self {
         Self {
+            binary: BINARY.to_string(),
             packages: HashSet::new(),
         }
     }
+
+    fn get_switches_runtime(&self) -> Switches {
+        match self.get_binary() {
+            "pip" => &["list", "--format", "json", "--not-required", "--user"],
+            "pipx" => &["list", "--json"],
+            _ => panic!("Cannot use {} for package management in python. Please use a valid package manager like pip or pipx", self.get_binary()),
```

Comment:
Please extract the message as a constant as far as possible. This message appears four times in this module.

Reply:
Done!